### PR TITLE
Minor cryptographic explanations

### PR DESCRIPTION
**Pull request by emanjon**
#201

### CODE REVIEW

File: draft-ietf-lake-edhoc.md [hunk 1]
```diff
@@ -1169,16 +1171,15 @@ An attacker can also send faked message_2, message_3, message_4, or error in an
...
+If two nodes unintentionally initiate two simultaneous EDHOC message exchanges with each other even if they only want to complete a single EDHOC message exchange, they MAY terminate the exchange with the lexicographically smallest G_X. If the two G_X values are equal, the received message_1 MUST be discarded to mitigate reflection attacks. Reflection attacks are not possible when the other endpoint's identity is verified, but can be a problem with trust on first use (TOFU) and could be a problem if methods with pre-shared keys are are specified in the future. Note that in the case of two simultaneous EDHOC exchanges where the nodes only complete one and where the nodes have different preferred cipher suites, an attacker can affect which of the two nodes’ preferred cipher suites will be used by blocking the other exchange.
```
Comment (pbtgit) [inline draft-ietf-lake-edhoc.md]: "An attacker can affect which of the two nodes’ preferred cipher suites will be used by blocking the other exchange."
What are the exact pre-conditions for such an attack?

Comment (gselander) [reply draft-ietf-lake-edhoc.md]: I opened a separate issue #226

Comment (emanjon) [reply draft-ietf-lake-edhoc.md]: This is wrong, I don't know if it was true in an early version or if has always been wrong.

I tried to fix this in PR #231

**Comment by emanjon** [comment]
This could not be merges for some reason so I made a separate commit 678619c. The commit also includes the fix to Paters comment above.

**Pull request closed without merging**